modules/apps/knowledge-base/knowledge-base-web/test/admin/js/components/LockedKBArticleModal.js

```diff
@@ -39,8 +41,24 @@ describe('LockedKBArticleModal', () => {
 		expect(queryByText('article-in-edition')).not.toBeInTheDocument();
 	});
 
-	it('renders the modal when try to edit/expire/delete a locked article', () => {
-		const {getByText} = render(<LockedKBArticleModal open={true} />);
+	it('does not render the modal first as group admin', () => {
```

Comment:
I think this test is unneccesary. While _open={false}_ nothing will be rendered, doesn't matter the `groupAdmin` value. And this is already covered by the first test.
And when both parameters are `true`, you are already testing `queryByText('article-in-edition-by-user-x-description')` in your next test **'renders the modal as group admin when try to edit/expire/delete a locked article'**